worker/worker-manager/score-worker-manager-impl/src/test/java/org/openscore/worker/management/monitor/ScheduledWorkerLoadMonitorTest.java

```diff
@@ -0,0 +1,108 @@
+package org.openscore.worker.management.monitor;
+
+import static org.junit.Assert.*;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.openscore.engine.node.services.WorkerNodeService;
+import org.openscore.worker.management.WorkerConfigurationService;
+import org.openscore.worker.management.services.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.io.Serializable;
+import java.util.HashMap;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.reset;
+import static org.mockito.Mockito.when;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = ScheduledWorkerLoadMonitorTest.MyTestConfig.class)
+public class ScheduledWorkerLoadMonitorTest {
+    @InjectMocks

```

Comment:
I don't think that you need the inject mocks annotation if you're using spring runner and mocking in the configuration class